pages/alerts.rst

```diff
@@ -312,6 +314,13 @@ Configuration Color
 Custom Message
     The message that will be sent to Slack.  The data described above can be used in this template.
 
+Backlog Message Items (optional)
+    The custom message has a data point named `backlog <http://localhost:8000/pages/alerts.html?highlight=alerts#backlog>`__, which gives
+    the preview of the messages that led to a slack notification.
+    Backlog Message Items will be used to override the Event definition Notification settings field `Message Backlog`
+    to limit the number of backlog messages sent as part of the slack message.
+    A Slack notification cannot have more than 50 backlog messages.
```

Comment:
Open discussion on the other PR, but I don't think it makes any sense to enforce a max on the limit value.